salt/metalk8s/repo/Ubuntu.sls

```diff
@@ -0,0 +1,30 @@
+Install python-apt:
+  pkg.installed:
+    - name: python-apt 
+
+
+Configure Kubernetes repository:
+  pkgrepo.managed:
+    - humanname: kubernetes
+    - name: deb http://apt.kubernetes.io/ kubernetes-xenial main
+    - file: /etc/apt/sources.list.d/kubernetes.list
+    - key_url: https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    - require:
+      - pkg: Install python-apt
+
+
+
+Configure Salt repository:
+  pkgrepo.managed:
+    - humanname: saltstack
+    - name: deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/archive/2018.3.4 bionic main
+    - file: /etc/apt/sources.list.d/saltstack.list
```

Comment:
Merge all repos we set up in the same list file (IMHO).

Reply:
Do the same as what I done for /etc/apt/sources.list.d/kubernetes.list I put all the source list in metalk8s.list